lib/login_gov/hostdata.rb

```diff
@@ -1,7 +1,9 @@
+require "json"
 require "login_gov/hostdata/ec2"
+require "login_gov/hostdata/log_formatter"
 require "login_gov/hostdata/s3"
 require "login_gov/hostdata/version"
-require "json"
+require "login_gov/hostdata/railtie" if defined?(Rails::Railtie)
```

Comment:
Question 1:

Should we auto-require like this? It's common for other gems to auto-require their railtie, but I don't like how they mysteriously reconfigure Rails apps by just including a gem

Alternative I think we should consider:
1. remove this line
2. apps would have to explicitly `require "login_gov/hostdata/railtie"` on their own

Reply:
I vote 2